cpp/recognition/CographRecognition.cpp

```diff
@@ -0,0 +1,58 @@
+#include <list>
+#include "graph/GraphTools.hpp"
+#include "recognition/CorneilStewartPerlCographRecognition.h"
+
+namespace Koala {
+    CorneilStewartPerlCographRecognition::CorneilStewartPerlCographRecognition(NetworKit::Graph &graph) : graph(graph),
```

Comment:
Probably more convenient line breaking would be:
```
CorneilStewartPerlCographRecognition::CorneilStewartPerlCographRecognition(
    NetworKit::Graph &graph) : graph(graph), is_cograph(State::UNKNOWN) {
```

Reply:
fixed